app/main/browser.js

```diff
@@ -36,19 +36,21 @@ import * as hotkeys from './hotkeys';
 import * as menu from './menu/index';
 import * as instrument from '../services/instrument';
 import registerAboutPages from './about-pages';
+import * as spawn from './spawn';
 import * as BW from './browser-window';
-import userAgentClient from '../shared/user-agent-client';
-
-import * as endpoints from '../shared/constants/endpoints';
+import UserAgentClient from '../shared/user-agent-client';
 
 const app = electron.app; // control application life.
 const ipc = electron.ipcMain;
 const globalShortcut = electron.globalShortcut;
-const userAgentPromise = userAgentClient.connect();
+const userAgentClient = new UserAgentClient();
 
 const appStartupTime = Date.now();
 instrument.event('app', 'STARTUP');
 
+// Start the UA service running on a different process
+spawn.userAgentService();
```

Comment:
Yeah they'll probably just fail since they can't use the port.

Tried it out with two Tofinos (or Twofinos), and It Just Works:tm: